spec/javascripts/Dashboard/ApiFilter.spec.jsx

```diff
@@ -1,45 +1,35 @@
+// @flow
+
 import React from 'react'
 import Enzyme, { mount } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 import { ApiFilter } from 'Dashboard/components/ApiFilter'
 
 Enzyme.configure({ adapter: new Adapter() })
 
-function getWrapper (apis = [], displayApis) {
-  return mount(<ApiFilter apis={apis} displayApis={displayApis} />)
-}
-
-let apiFilter
-
 const apis = [
-  { service: { name: 'api 0' } },
-  { service: { name: 'api 1' } },
-  { service: { name: 'api 2' } }
+  { id: 0, name: 'api 0', link: '', type: 'backend' },
+  { id: 1, name: 'api 1', link: '', type: 'backend' },
+  { id: 11, name: 'api 11', link: '', type: 'backend' }
 ]
+const domClass = 'class'
 
-beforeEach(() => {
-  apiFilter = getWrapper()
-})
+let apiFilter
 
-afterEach(() => {
-  apiFilter.unmount()
+beforeEach(() => {
+  apiFilter = mount(<ApiFilter apis={apis} domClass={domClass} />)
 })
 
 it('should render itself', () => {
   expect(apiFilter.find('.ApiFilter').exists()).toBe(true)
 })
 
-it('should filter APIs passed in props by name', () => {
-  const displayApis = jest.fn()
-  apiFilter = getWrapper(apis, displayApis)
-
+it.skip('should filter APIs passed in props by name', () => {
```

Comment:
I had to take this out of the equation, since we no longer pass a displayApis function. Can't be spied since it's neither a class component nor it is an exported function.
I can't think of any other option rather than passing the argument or exporting the method, wdyt @didierofrivia ?

Reply:
This test, besides being skipped, is doing nothing, I think we can remove it by now.
You can create a new task to improve this test suite later

Reply:
Approving to not block you, but try to create a task for this